src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -100,7 +100,7 @@ internal static Collection<FunctionBinding> GetBindings(ScriptHostConfiguration
                         case BindingType.HttpTrigger:
                             bindings.Add(new HttpBinding(config, bindingMetadata, FileAccess.Read));
                             break;
-                        case BindingType.EasyTable:
+                        case BindingType.MobileTable:
                             EasyTableBindingMetadata easyTableMetadata = (EasyTableBindingMetadata)bindingMetadata;
```

Comment:
Why not just go all the way in the Script repo and rename all the artifacts in here, including the binding classes, etc.? Heck, I don't know why you don't just rename the extension now, pull that in and do it all in one go. Should take about an hour.